data/tmpl/values/content5-snippet.html

```diff
@@ -1,4 +1,4 @@
 <section id="allaboard_snippet" class="allaboard-snippet">
   <p>You’re different, that’s why we like you. Keep up with issues surrounding the Web.</p>
-  <a href="https://www.mozilla.org/#newsletter-subscribe" id="allaboard_anippet_intent" class="button">Stay informed</a>
+  <a href="https://www.mozilla.org/#newsletter-subscribe?utm_source=fxonboarding&amp;utm_medium=snippet&amp;utm_campaign=onboardingv1&amp;utm_content=community" id="allaboard_anippet_intent" class="button">Stay informed</a>
```

Comment:
@schalkneethling the bookmark in the URL needs to be after the query parameters. like:

https://www.mozilla.org/en-US/?utm_source=fxonboarding&utm_medium=snippet&utm_campaign=onboardingv1&utm_content=community#newsletter-subscribe